Snowflake.Data.Tests/UnitTests/Logger/EasyLoggerManagerTest.cs

```diff
@@ -0,0 +1,110 @@
+/*
+ * Copyright (c) 2023 Snowflake Computing Inc. All rights reserved.
+ */
+
+using System.IO;
+using System.Linq;
+using NUnit.Framework;
+using Snowflake.Data.Configuration;
+using Snowflake.Data.Core;
+using Snowflake.Data.Log;
+
+namespace Snowflake.Data.Tests.UnitTests.Logger
+{
+    [TestFixture, NonParallelizable]
+    public class EasyLoggerManagerTest
+    {
+        
+        private const string InfoMessage = "Easy logging Info message";
+        private const string DebugMessage = "Easy logging Debug message";
+        private const string WarnMessage = "Easy logging Warn message";
+        private const string ErrorMessage = "Easy logging Error message";
+        private const string FatalMessage = "Easy logging Fatal message";
```

Comment:
Documentation states what are log levels possible to configure. It is the common list for all drivers. TRACE is one of the options.

However our implementation for logger do not have loggger.Trace() method. So in our case trace is almost like all level. Do you think it would be good to add Trace() method to our logger implementation?

Fatal is not defined by easy logging documentation, but from the other side what's wrong with include also fatal message in tests since it is one of levels which our logger implementation supports?

Reply:
That is a very good question indeed. On one hand support would expect to have same "enum" value for all loggers - especially if we have `common` section and log level there. On the other hand, if we have .NET developers who are accustomed to .NET log levels, it may be strange for using another log levels. We can discuss it across our team.

Reply:
From my point of view, let's say I will add Trace() method to the logger. But no of the driver classes logs with Trace log level. So it will have no effect on anything. It's just an enhancement for the future.

Reply:
And what should happen if support configures `TRACE` in a config file? I see it like this:
1. If `TRACE` is configured - everything is logged.
2. Debug, info, warn, error - straightforward.
But in that case error and fatal messages are logged in the same manner. But probably this is the best we can get.

Reply:
as we discussed, we will leve it as it is